modules/swagger-codegen/src/main/resources/objc/model-body.mustache

```diff
@@ -13,7 +13,11 @@
   }
   return self;
 }
-
+{{#vars}}{{^vendorExtensions.x-unique-id-key}}{{/vendorExtensions.x-unique-id-key}}{{#vendorExtensions.x-unique-id-key}}
+-(NSString*)indexPropertyName {
+    return @"{{name}}";
+}
+{{/vendorExtensions.x-unique-id-key}}{{/vars}}
```

Comment:
If this vendor extension is objc-specified, shall we name it as `x-objc-unique-id-key` so that it's immediately clear to the others this vendor extension is used by objc client generator?